libs/hooks/useUpkeeps/index.tsx

```diff
@@ -48,7 +48,7 @@ export const useUpkeeps: (network: KnownNetwork | undefined) => Record<string, U
             const now = Math.floor(Date.now() / 1000);
             const from = now - (poolInfo?.updateInterval || ONE_HOUR) * 2;
 
-            const rawUpkeeps = await fetch(`${POOLS_API}/upkeeps?network=${network}&from=${from}`)
+            const rawUpkeeps = await fetch(`${TRACER_API}/pools/upkeeps?network=${network}&from=${from}`)
```

Comment:
we don't have an upkeeps endpoint for v2, was keen to just use the graph (because the api syncs it from the graph for v1 anyway)

Reply:
ok, will create a ticket for this, was just changing whatever was existing to fit new type